src/Microsoft.AspNetCore.Mvc.ViewFeatures/Rendering/HtmlHelperFormExtensions.cs

```diff
@@ -47,7 +47,7 @@ public static MvcForm BeginForm(this IHtmlHelper htmlHelper)
         /// If <c>true</c>, &lt;form&gt; elements will include an antiforgery token.
         /// If <c>false</c>, suppresses the generation an &lt;input&gt; of type "hidden" with an antiforgery token.
         /// If <c>null</c>, &lt;form&gt; elements will include an antiforgery token only if
-        /// <paramref name="method"/> is not <see cref="FormMethod.Get"/>.
+        /// HTTP method is not <see cref="FormMethod.Get"/>.
```

Comment:
Where the method does not have a `method` parameter, the right thing to do is remove `if <paramref name="method"/> is not <see cref="FormMethod.Get"/>` entirely. Here for example, will always include the token when `antiforgery` is `null`.